### DISCUSSION TITLE
Revising gitignore file

### DISCUSSION THREAD

**Discussion by hsynlms** [fastify/fastify#3087] [General] [answered]
I think `gitignore` file needs to be revised. Environment-specific files should not be ignored in the project's gitignore file. Instead, they all should be ignored in the global gitignore file. That's just my suggestion.

Your thoughts?

### Example

The current gitignore file contains:

```
# editor files
.vscode
.idea
```

**Comment by jsumners** [comment, marked as answer]
No, we explicitly added those because we do not want contributors at large adding them to the project.

**Reply by hsynlms** [reply]
I know those lines added on purpose. I just proposed that those are environment-specific files and we can tell contributors to exclude those files in commits via the global gitignore file.

**Reply by jsumners** [reply]
We don't have to constantly battle them being checked in if we ignore them in the project.

**Reply by hsynlms** [reply]
Okay.
